tests/appsec/test_automated_login_events.py

```diff
@@ -36,6 +36,8 @@ class Test_Login_Events:
     BASIC_AUTH_INVALID_USER_HEADER = "Basic aW52YWxpZFVzZXI6MTIzNA=="  # base64(invalidUser:1234)
     BASIC_AUTH_INVALID_PASSWORD_HEADER = "Basic dGVzdDoxMjM0NQ=="  # base64(test:12345)
 
+    MANUAL_KEEP = 2
```

Comment:
```suggestion
    MANUAL_KEEP_SAMPLING_PRIORITY = 2
```

Maybe it's a bit more informative.

Reply:
fixed in https://github.com/DataDog/system-tests/pull/1347/commits/c4d6c8c814f7842d98dddeab2638c8927bed378e